src/main/java/org/sagebionetworks/bridge/udd/helper/SesHelper.java

```diff
@@ -57,6 +85,26 @@ public final void setSesClient(AmazonSimpleEmailServiceClient sesClient) {
      */
     public void sendPresignedUrlToAccount(StudyInfo studyInfo, PresignedUrlInfo presignedUrlInfo,
             AccountInfo accountInfo) {
+        String presignedUrlStr = presignedUrlInfo.getUrl().toString();
+        String expirationTimeStr = presignedUrlInfo.getExpirationTime().toString();
+        String bodyHtmlStr = String.format(BODY_TEMPLATE_HTML, presignedUrlStr, expirationTimeStr);
+        String bodyTextStr = String.format(BODY_TEMPLATE_TEXT, presignedUrlStr, expirationTimeStr);
+        Body body = new Body().withHtml(new Content(bodyHtmlStr)).withText(new Content(bodyTextStr));
+
+        sendEmailToAccount(studyInfo, accountInfo, body);
+    }
+
+    /**
+     * Helper method, which sends the given email message body to the given account
+     *
+     * @param studyInfo
+     *         study info, used to construct the email message, must be non-null
+     * @param accountInfo
+     *         account to send the email to, must be non-null
+     * @param body
+     *         email message body
+     */
+    private void sendEmailToAccount(StudyInfo studyInfo, AccountInfo accountInfo, Body body) {
         // from address
```

Comment:
Email sending code may someday make for a good shared library, if we can factor it out appropriately.